spec/lib/reporting_spec.rb

```diff
@@ -30,6 +34,10 @@
     user1.notes.create(body: '')
   end
 
+  let(:nil_note) do
+    user1.notes.create(body: nil)
+  end
+
   let(:whitespace_note) do
```

Comment:
You start the spec with many `let` blocks and then many `context` blocks. This one for example is only relevant in a single context so there is no benefit for it have floated to the top and be available to all. Better that it sit within the relevant context thereby better describing what that context needs. I don't think we need bother asserting the body of the note either.

Reply:
[d2d4833](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/645/commits/d2d483327bb4df7ec1627bfe1d0b6aa15dcb408f)